src/blocks/folder-blocks/infinite-canvas/index.tsx

```diff
@@ -11,7 +11,7 @@ import { Button } from "@primer/react";
 
 const width = 5000;
 const height = 5000;
-const defaultDimensions = [200, 100];
+const defaultDimensions: [number, number] = [200, 100];
```

Comment:
cleaned up some type errors along the way

Reply:
for posterity and my own edification

[From slack](https://github.slack.com/archives/C01JU2DH6V7/p1652377727671269?thread_ts=1652377642.798219&cid=C01JU2DH6V7):
```
if you write just [200, 100] typescript infers number[] as the type of defaultDimensions. 
```